archetypes/pom.xml

```diff
@@ -22,6 +22,7 @@
         <module>client</module>
         <module>embedded</module>
         <module>store</module>
+        <module>servertask</module>
```

Comment:
Sorry I didn't make this clearer before, can you also update the directory to `server-task`.